### PR TITLE
updates chart for new universal redis client

### LINKED ISSUE #68: Chart has not been updated to reflect new configuration option names.
At the very least this includes things like `addrs` instead of `hosts` for redis.. Maybe other things.

### PR DESCRIPTION
**Pull request by asoorm**
resolves #68

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -89,19 +97,19 @@ tib:
...
     profiles: tyk-tib-profiles-conf
 
 dash:
   replicaCount: 1
   hostName: "localhost"
-  license: ""
+  license: "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhbGxvd2VkX25vZGVzIjoiOGU4NWFjZTUtNGRjOC00OTk1LTRkNzUtNWI4YTM2YzA4ZTJmLDViMDAzYzI1LTg3ZjYtNDIyZC00ODBhLWRiMGNjZDdhNjBlYyw2NWM4NDhhMy0xYWU4LTRhNzQtNGYwOS05ZjA5YmNkYjM5MGQsZmIwMTZhM2EtZjAxYi00ZWJiLTRlYzQtODI0NjkzYzE0ZWJmLDBkMDYwYTQ3LWUwMTktNDU1My01MGFhLWVjOWY4ZTEzZGE3YiwxNzdjMmZiZC04ZjRiLTQxZjEtNTg2ZC01MGE5YjQ0M2UzZWQsYjZjNGVjN2QtZGMwMi00NTg4LTcyMGMtY2FmNjRlNDEzYzE0LDNhMmJkYTBkLWY0M2ItNGY3OS00YzkzLWJkNDcxOTAzZDRiYSxlYWM5NDRlZC02NmVhLTRkNmUtN2IwYi05YmQ4N2VhZGRmNTQsZWJiNDk4MzQtOTFiNi00OTE5LTdkOTYtNWFkN2Y2ZjM0MTBiIiwiZXhwIjoxNjAxODMyOTMxLCJpYXQiOjE1OTkyNDA5MzEsIm93bmVyIjoiNTc3OTcxMTk0NWY5MmU2Njg5MDAwMTI3Iiwic2NvcGUiOiJtdWx0aV90ZWFtLHJiYWMsZ3JhcGgiLCJ2IjoiMiJ9.LnxOKxc_qVztRZ5cJM0Orog3-_eeco-caTy_QrbAvjdo_4G2r9Odk1KuGmLrW9fsjJZ-siALRb33DgPSQazEIaIgbVGbzp-r024Hc5mQuwZp2sZ9hBnxgra4xxvETLaUs0U9HllX7FVU2DIXVSzNq2HgkiVGYol7BS8u5mdxr9HqssSq7VZ6tOpFnJyZonHzm7E-PO_CfEp3hHIRLo6oLPbCoY5iO2MiqYWljox5g2kSVQo7_PQBCxyO1lMbiHYCW6_s6uMHPSqGiB_td8-ecX9CgjXnTPdd1DR2H4c7hnfKSyoGRGFsZsdDmdcawKXghrxLp9DUK2VgKCR8KVOsNw"
```
Comment (vesko-tyk) [inline values.yaml:112]: @asoorm Could you remove this please? The license should not be exposed here. Thanks

**Review by excieve — CHANGES_REQUESTED** [review]
Nice, thanks! I'd suggest using a YAML list where variable is logically always a list of items for better usability.

File: values.yaml [hunk 2]
```diff
@@ -12,13 +12,21 @@ secrets:
   APISecret: "CHANGEME"
   AdminSecret: "12345"
 
+# bitnami redis example
 redis:
     shardCount: 128
-    host: "tyk-redis-master.tyk-ingress.svc.cluster.local"
-    port: 6379
+    addrs: "tyk-redis-master.tyk-ingress.svc.cluster.local:6379"
```
[the remark this answers is not in the document]
Comment (drpebcak) [reply values.yaml:18]: I agree the list is easier, if you are using values.yaml. If you are using `--set`, it becomes a pain to access elements of the list.

[the remark this answers is not in the document]
Comment (drpebcak) [reply values.yaml:18]: (personally I use values.yaml, so I'm all for that)

[the remark this answers is not in the document]
Comment (joshblakeley) [reply values.yaml:18]: Thanks Taylor - i'm willing to ignore the --set crowd also

**Comment by drpebcak** [comment]
Any movement on this?

**Comment by asoorm** [comment]
> Any movement on this?

Hello - I will try to make changes today.

**Comment by asoorm** [comment]
@excieve PTAL

**Comment by asoorm** [comment]
After further testing, I found a couple of issues with erroneous log messages:

```
time="Nov 08 08:52:25" level=error msg="Error trying to get TTL: MOVED 609 10.244.0.60:6379"
time="Nov 08 08:52:26" level=error msg="Redis health check failed" error="storage: Redis is either down or ws not configured" liveness-check=true prefix=main
time="Nov 08 08:52:34" level=error msg="Error trying to get TTL: MOVED 609 10.244.0.60:6379"
time="Nov 08 08:52:36" level=error msg="Redis health check failed" error="storage: Redis is either down or ws not configured" liveness-check=true prefix=main
time="Nov 08 08:52:40" level=error msg="Error trying to get TTL: MOVED 609 10.244.0.60:6379"
time="Nov 08 08:52:40" level=error msg="Could not EXPIRE key: MOVED 609 10.244.0.60:6379"
```

Moving to draft till I figure this out.

**Comment by vesko-tyk** [comment]
This is now implemented in this PR:https://github.com/TykTechnologies/tyk-helm-chart/pull/73

**Pull request closed without merging**